smithy-rules-engine/src/test/java/software/amazon/smithy/rulesengine/language/IntegrationTest.java

```diff
@@ -15,8 +15,7 @@
 
 package software.amazon.smithy.rulesengine.language;
 
-import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
This should stay as individual imports instead of a `*`, adding the other assertion methods as needed.

Reply:
ugh, intellij being too helpful here. reverted